pyro/distributions/torch.py

```diff
@@ -67,6 +67,19 @@ def __init__(self, ps, n=1):
         super(Multinomial, self).__init__(n, probs=ps)
 
 
+class MultivariateNormal(torch.distributions.MultivariateNormal, TorchDistributionMixin):
+    def __init__(self, loc, cov):
```

Comment:
Can we omit the `__init__` method and replace `cov` with `covariance_matrix` (rather than replace `covariance_matrix` with `cov` as I see you've done in a few tests)?

Reply:
There are two reasons to omit the `__init__` method:
- this saves us a renaming later on
- we will have immedate access to the `scale_tril` arg, which we want in a few places

Reply:
My bad..I thought that we are using `cov` in `torch.distributions`, but I was mistaken. Will revert.

Reply:
Yeah, I'm fixing the broken `MultivariateNormal` docs upstream right now :smile:

Reply:
> we will have immedate access to the scale_tril arg, which we want in a few places

Makes sense. Removed.